src/Illuminate/Redis/Connections/PredisClusterConnection.php

```diff
@@ -13,8 +14,12 @@ class PredisClusterConnection extends PredisConnection
      */
     public function flushdb()
     {
+        $command = class_exists(ServerFlushDatabase::class)
+            ? ServerFlushDatabase::class
+            : FLUSHDB::class;
+
         $this->client->executeCommandOnNodes(
```

Comment:
This method doesn't exist in 2.0 either 😬

Reply:
@tillkruss what alternative should be used?

Reply:
I mentioned that in #45530, you can just loop over the `$this->client`.